src/explore-education-statistics-frontend/src/services/dataSetFileService.ts

```diff
@@ -86,6 +89,7 @@ export const dataSetFileFilters = [
 export type DataSetFileFilter = (typeof dataSetFileFilters)[number];
 
 export interface DataSetFileListRequest {
+  dataSetType?: 'all' | 'api';
```

Comment:
We could extract out a `DataSetType` for this and use it across the relevant components and props. Seems like we're currently using strings instead (which isn't type safe).